docs/components/DataList/DataList.stories.mdx

```diff
@@ -252,7 +252,7 @@ children and inserts it above the list of items.
 
 You can add a button that will clear all the filters by using the combination of
 the `Combobox` and `Button` components (check out the `Code` tab in the
-["Clear All Filters" story](../?path=/story/components-lists-and-tables-datalist-web--clear-all-filters)
+["Clear All Filters" story](../?path=/code/components-lists-and-tables-datalist-web--clear-all-filters)
```

Comment:
I think this is more of an issue with the `Playground` component, rather than with Storybook. I will revert this change for now, so that users could at least land on the docs page.